packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -61,6 +61,8 @@ class PerformanceController
   ValueListenable<List<FlutterFrame>> get flutterFrames => _flutterFrames;
   final _flutterFrames = ValueNotifier<List<FlutterFrame>>([]);
 
+  final _flutterFramesById = <String, FlutterFrame>{};
```

Comment:
The changes in this file are duplicated from performance/legacy/performance_controller.dart. These two files will diverge soon once the files under performance/ are migrated to use the FrameTiming API for flutter frames, though changes like this will continue to need to be applied to both files until the changes we need to use the FrameTiming API hit flutter stable.